src/test/resources/application.yml

```diff
@@ -1,3 +1,10 @@
 clients:
     users-api-host: http://test.com/users
     teams-api-host: http://test.com/teams
+
+retry:
```

Comment:
We can improve client calls in order to be a fault tolerant service with spring-retry for 5XX http status response errors.